jetty-server/src/main/config/etc/jetty-ssl-context.xml

```diff
@@ -11,12 +11,24 @@
 
 <Configure id="sslContextFactory" class="org.eclipse.jetty.util.ssl.SslContextFactory$Server">
   <Set name="Provider"><Property name="jetty.sslContext.provider"/></Set>
-  <Set name="KeyStorePath"><Property name="jetty.base" default="." />/<Property name="jetty.sslContext.keyStorePath" deprecated="jetty.keystore" default="etc/keystore"/></Set>
+  <Set name="KeyStorePath">
+    <Property name="jetty.sslContext.keyStoreAbsolutePath">
+      <Default>
+        <Property name="jetty.base" default="." />/<Property name="jetty.sslContext.keyStorePath" deprecated="jetty.keystore" default="etc/keystore"/>
+      </Default>
+    </Property>
+  </Set>
   <Set name="KeyStorePassword"><Property name="jetty.sslContext.keyStorePassword" deprecated="jetty.keystore.password" default="OBF:1vny1zlo1x8e1vnw1vn61x8g1zlu1vn4"/></Set>
   <Set name="KeyStoreType"><Property name="jetty.sslContext.keyStoreType" default="JKS"/></Set>
   <Set name="KeyStoreProvider"><Property name="jetty.sslContext.keyStoreProvider"/></Set>
   <Set name="KeyManagerPassword"><Property name="jetty.sslContext.keyManagerPassword" deprecated="jetty.keymanager.password" default="OBF:1u2u1wml1z7s1z7a1wnl1u2g"/></Set>
-  <Set name="TrustStorePath"><Property name="jetty.base" default="." />/<Property name="jetty.sslContext.trustStorePath" deprecated="jetty.truststore" default="etc/keystore"/></Set>
+  <Set name="TrustStorePath">
+    <Property name="jetty.sslContext.trustStoreAbsolutePath">
+      <Default>
+        <Property name="jetty.base" default="." />/<Property name="jetty.sslContext.trustStorePath" deprecated="jetty.truststore" default="etc/keystore"/>
+      </Default>
+    </Property>
+  </Set>
```

Comment:
Rather than introduce a new property to resolve the absolute vs relative issue, might it not be better to introduce a static helper method:
```suggestion
  <Set name="TrustStorePath">
    <Call class="org.eclipse.jetty.xml.XmlConfiguration" name="addPaths">
      <Arg><Property name="jetty.base" default="." /></Arg>
      <Arg><Property name="jetty.sslContext.trustStorePath" deprecated="jetty.truststore" default="etc/keystore"/></Arg>
    </Call>
  </Set>
```
Perhaps even URIUtil.addPaths can be used?    Hmmm but then I guess we get caught up in portability issues and working out if `C:some\path` is relative or not.